controllers/base_controller.py

```diff
@@ -117,6 +117,17 @@ def _require_login(self, target_url="/"):
                 abort=True
             )
 
+    def _require_permission(self, permission):
+        self._require_login()
+        logging.info("logged in")
+        self._require_registration()
+        logging.info("registered")
+        if permission not in self.user_bundle.account.permissions:
+            return self.redirect(
```

Comment:
I agree this should be improved, let's address it in a future pull. A common "you don't have permission to do that, here's how to ask for permission" screen seems good.